netbox_librenms_plugin/views/base/librenms_sync_view.py

```
@@ -268,6 +330,61 @@ def get_vlan_context(self, request, obj):
         """
         return None
 
+    def get_module_context(self, request, obj):
+        """
+        Get the context data for module sync.
+        Subclasses should override this method if applicable.
+        """
+        return None
+
+    @staticmethod
+    def _build_all_server_mappings(obj, active_server_key):
```

Flake8 (7.3.0)

[error] redefinition of unused '_build_all_server_mappings' from line 127

(F811)